Description: Downgrade the assert_cmd crate depdendency
 Currently, only the 2.0.12 version of the assert_cmd is available in unstable.
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
--- rust-pastel-0.10.0.orig/Cargo.toml
+++ rust-pastel-0.10.0/Cargo.toml
@@ -81,7 +81,7 @@ version = "1.10"
 version = "0.5.0"
 
 [dev-dependencies.assert_cmd]
-version = "2.0.13"
+version = "2.0.12"
 
 [dev-dependencies.criterion]
 version = "0.5"
